src/status_im/contexts/wallet/swap/events.cljs

```diff
@@ -12,11 +13,9 @@
             (assoc-in [:wallet :ui :swap :asset-to-pay] token)
             (assoc-in [:wallet :ui :swap :network] network))
     :fx [(if network
-           [:dispatch
-            [:toasts/upsert
-             {:id   :swap-error
-              :type :negative
-              :text "Not implemented yet"}]]
+           [:dispatch-n
```

Comment:
Quick one I missed, `:dispatch-n` is deprecated. `:fx` supersedes it and supports multiple sequential dispatches.

Reply:
You didn't miss it the first time. I added it when I rebased. I removed the use of `dispatch-n`. Thanks for the comment.